src/Adapters/ApproversCategoriesLookup.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace ProfessionalWiki\PageApprovals\Adapters;
+
+use ProfessionalWiki\PageApprovals\Application\ApproversCategoriesLookupInterface;
+use Wikimedia\Rdbms\IDatabase;
+
+class ApproversCategoriesLookup implements ApproversCategoriesLookupInterface {
```

Comment:
We don't name interfaces `...Interface`. Since this is a Database implementation, as opposed to something like InMemory or other test double, the class should be called `DatabaseApproversCategoriesLookup` and the interface `ApproversCategoriesLookup`. Look at the other classes in Adapters.